anta/cli/nrfu/utils.py

```diff
@@ -65,9 +65,11 @@ def run_tests(ctx: click.Context) -> None:
         ctx.exit()
 
 
-def _get_result_manager(ctx: click.Context) -> ResultManager:
+def _get_result_manager(ctx: click.Context, *, hide: bool = True) -> ResultManager:
```

Comment:
Let's be more explicit:
```suggestion
def _get_result_manager(ctx: click.Context, *, apply_hide_filter: bool = True) -> ResultManager:
```

Reply:
Updated